src/export/FSHExporter.ts

```diff
@@ -219,9 +219,13 @@ export class FSHExporter {
   private groupByProfile(): Map<string, Exportable[]> {
     const files: Map<string, Exportable[]> = new Map();
 
-    // Group profiles and examples of those profiles into individual files
-    this.fshPackage.profiles.forEach(profile => {
-      files.set(`${profile.name}.fsh`, [profile]);
+    // Group profiles, logicals, and resources with their respective examples into individual files
+    [
+      ...this.fshPackage.profiles,
+      ...this.fshPackage.logicals,
```

Comment:
This is a bit of a tough call, because a `Logical` cannot have `Instances` made of it, but it can have `Invariants`. So I could go either way on if it should have its own file, or just be in one big `logicals.fsh` file. But if you do want to keep each `Logical` in its own file, I think we will want to remove `Logicals` from the logic that finds out if there are any instances of a `Logical`, since that isn't possible.

Reply:
My opinion is that since a Logical can have invariants on it, each one should be allowed to have its own file in the `group-by-profile` strategy. I'll update the strategy to skip checking for Instances of a Logical.

Reply:
Upon a closer read, no code change is necessary, but I have updated the comments to be more accurate.